pkg/autoscaler/metrics/collector.go

```diff
@@ -27,17 +27,13 @@ import (
 	"knative.dev/pkg/logging/logkey"
 	av1alpha1 "knative.dev/serving/pkg/apis/autoscaling/v1alpha1"
 	"knative.dev/serving/pkg/autoscaler/aggregation"
+	"knative.dev/serving/pkg/autoscaler/config"
 )
 
 const (
 	// scrapeTickInterval is the interval of time between triggering StatsScraper.Scrape()
 	// to get metrics across all pods of a revision.
 	scrapeTickInterval = time.Second
-
-	// BucketSize is the size of the buckets of stats we create.
-	// NB: if this is more than 1s, we need to average values in the
-	// metrics buckets.
-	BucketSize = scrapeTickInterval
```

Comment:
The BucketSize change looks like another nice and non-contentious change that could go into a separate PR imo.

Reply:
ack => https://github.com/knative/serving/pull/6838.